src/components/charts/list/DiscoverCharts.tsx

```diff
@@ -617,20 +625,27 @@ export function ChartGroupListMin({
         )
     }
     return (
-        <div className="chart-group" style={{ minHeight: '280px' }}>
+        <div className="chart-group">
             <div className="chart-group__header">
-                <div className="flexbox">
+                <div className="">
                     <h2 className="chart-grid__title">Chart Groups</h2>
-                    <button
-                        type="button"
-                        className="chart-group__view-all"
-                        onClick={(e) => history.push(match.url + '/group')}
-                    >
-                        View All
-                    </button>
+                    <p className="mb-16 mt-4">
+                        Use chart groups to preconfigure and deploy frequently used charts together. Learn more about
+                        chart groups
+                    </p>
+                    <div className="flex content-space">
+                        {renderCreateGroupButton()}
+                        <div
+                            className="cb-5 fw-6 fs-13 flex fcb-5 cursor"
+                            onClick={(e) => history.push(match.url + '/group')}
```

Comment:
Can we try to avoid the inline function here on onClick by making a separate function for this

Reply:
Done